plugins/woocommerce-blocks/packages/prices/utils/test/price.js

```diff
@@ -5,24 +5,25 @@ import { formatPrice, getCurrency } from '../price';
-		${ 10000000000 }    | ${ '€' }  | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 3 }    | ${ '€10,000,000.000' }
-		${ 10000000000000 } | ${ '€ ' } | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 6 }    | ${ '€ 10,000,000.000000' }
-		${ 10000000 }       | ${ '€ ' } | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 0 }    | ${ '€ 10,000,000' }
-		${ 1000000099 }     | ${ '$' }  | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '$10,000,000.99' }
-		${ 1000000099 }     | ${ '$' }  | ${ '' }  | ${ '.' }          | ${ ',' }         | ${ 2 }    | ${ '$10.000.000,99' }
+		value               | prefix         | suffix   | thousandSeparator | decimalSeparator | minorUnit | expected
+		${ 1020 }           | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '€10.20' }
+		${ 1000 }           | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '€10.00' }
+		${ 1000 }           | ${ '' }        | ${ '€' } | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '10.00€' }
+		${ 1000 }           | ${ '' }        | ${ '$' } | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '10.00$' }
+		${ '1000' }         | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '€10.00' }
+		${ 0 }              | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '€0.00' }
+		${ '' }             | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '' }
+		${ null }           | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '' }
+		${ undefined }      | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '' }
+		${ 100000 }         | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '€1,000.00' }
+		${ 1000000 }        | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '€10,000.00' }
+		${ 1000000000 }     | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '€10,000,000.00' }
+		${ 10000000000 }    | ${ '€' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 3 }    | ${ '€10,000,000.000' }
+		${ 10000000000000 } | ${ '€ ' }      | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 6 }    | ${ '€ 10,000,000.000000' }
+		${ 10000000 }       | ${ '€ ' }      | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 0 }    | ${ '€ 10,000,000' }
+		${ 1000000099 }     | ${ '$' }       | ${ '' }  | ${ ',' }          | ${ '.' }         | ${ 2 }    | ${ '$10,000,000.99' }
+		${ 1000000099 }     | ${ '$' }       | ${ '' }  | ${ '.' }          | ${ ',' }         | ${ 2 }    | ${ '$10.000.000,99' }
+		${ 1000 }           | ${ '&#x62f;' } | ${ '' }  | ${ '.' }          | ${ ',' }         | ${ 2 }    | ${ 'د10,00' }
```

Comment:
Just wanted to ask a clarifying question as it seems this is the only test line that was added.

Not knowing that character `#x62f` is a RTL character I was expecting a "prefix" to be added to the beginning of the string. Wouldn't we be better off splitting that into two tests so we can confirm each individually:
1. Test that a prefix containing an entity is decoded
2. Test that a RTL entity is placed on the right side of the string

Also do we need to test other entity formats (like `&cent;`) or do we assume that decodeEntities handles all the testing for different variations?

Reply:
Thanks for the suggestions! :) I'll add the other test line.

> Also do we need to test other entity formats (like &cent;) or do we assume that decodeEntities handles all the testing for different variations?

I think two will be enough, one RTL one LTR. Otherwise, where would we draw the line? [all currency characters](https://github.com/woocommerce/woocommerce/blob/dev/unify-decodeEntity/plugins/woocommerce/tests/e2e-pw/data/settings.js#L10:L10), all possible characters?